pkgs/by-name/li/libitl/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libitl";
+  version = "0.1.8";
+
+  nativeBuildInputs = [ cmake ];
+  outputs = [ "out" ];
+
+  src = fetchFromGitHub {
+    owner = "arabeyes-org";
+    repo = "ITL";
+    rev = "cf4a5555f492c6293efa1f8880d090c9f1769509";
+    hash = "sha256-Hp+NXibSlF9LMRRlHWISAiQSz/dbMNjjEfKCNFVmIxo=";
+  };
+
+  buildPhase = ''
+    cmake .
+    make
+  '';
+
```

Comment:
```suggestion
```
I believe this can be removed, defaualt buildPhase already executes make/cmake.

Reply:
Done.